cmd/composer-cli/root/root.go

```diff
@@ -103,7 +104,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&cloudSocketPath, "cloudsocket", "", "/run/cloudapi/api.socket", "Path to the cloudapi server's socket file")
 	rootCmd.PersistentFlags().IntVar(&testMode, "test", 0, "Pass test mode to compose. 1=Mock compose with fail. 2=Mock compose with finished.")
 	rootCmd.PersistentFlags().IntVar(&httpTimeout, "timeout", 240, "Timeout to use for server communication. Set to 0 for no timeout")
-
+	rootCmd.PersistentFlags().BoolVarP(&weldrOnly, "weldr-only", "", false, "Only use the WELDR API")
```

Comment:
I'd add a hint on "the history" to make it more understandable?
```suggestion
	rootCmd.PersistentFlags().BoolVarP(&weldrOnly, "weldr-only", "", false, "Only use the WELDR API; avoids using the newer CloudAPI")
```